api/v1beta1/module_types.go

```diff
@@ -184,16 +184,16 @@ type DevicePluginSpec struct {
 	Volumes []v1.Volume `json:"volumes,omitempty"`
 }
 
-// ModuleSpec describes how the OOT operator should deploy a Module on those nodes that need it.
+// ModuleSpec describes how the KMM operator should deploy a Module on those nodes that need it.
 type ModuleSpec struct {
 	// DevicePlugin allows overriding some properties of the container that deploys the device plugin on the node.
-	// Name is ignored and is set automatically by the OOT Operator.
+	// Name is ignored and is set automatically by the KMM Operator.
 	// +optional
 	DevicePlugin *DevicePluginSpec `json:"devicePlugin"`
 
-	// DriverContainer allows overriding some properties of the container that deploys the driver on the node.
-	// Name and image are ignored and are set automatically by the OOT Operator.
-	DriverContainer DriverContainerSpec `json:"driverContainer"`
+	// ModuleLoader allows overriding some properties of the container that loads the kernel module on the node.
+	// Name and image are ignored and are set automatically by the KMM Operator.
+	ModuleLoader ModuleLoaderSpec `json:"driverContainer"`
```

Comment:
Why node change the JSON key as well?

Reply:
@yevgeny-shnaidman?

Reply:
actually missed that one, thanks for the catch @fabiendupont ! Will fix it in next commit

Reply:
It will also need a PR to update it in https://github.com/qbarrand/oot-operator/blob/main/api/v1beta1/module_types.go#L239, right?

Reply:
yes

Reply:
Updated with @yevgeny-shnaidman's second upstream PR.

Reply:
/lgtm